agent.go

```diff
@@ -380,6 +382,61 @@ func (s *sandbox) teardown() error {
 	return s.channel.teardown()
 }
 
+type initMount struct {
+	typ, src, des string
+	options       []string
+}
+
+func initAgentAsInit() error {
+	agentLog.Infof("initAgentAsInit(), agent version: %s\n", version)
+
+	mounts := []initMount{
```

Comment:
I'd be better to define this outside of this function. This should be const structure list.